src/guide/index.jade

```diff
@@ -23,7 +23,7 @@ html(lang='en')
 
     //- Icons
     link(
-      href = "//smaatoui.github.io/icon-library/css/icons.css"
+      href = "//s3.amazonaws.com/smt-ui-icon-library/icons.css"
```

Comment:
Thinking if it should be hosted on smaato.github.io instead of S3.

Reply:
This is only a temporary solution.